### ISSUE TITLE
Swiper thumbs slider is not navigable by clicking if added after parent slider initialization

### ISSUE THREAD

**Issue by Chaitanya-Raj** [nolimits4web/swiper#6873]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://codesandbox.io/p/sandbox/swiper-thumbs-gallery-forked-kqflj2?file=%2Findex.html%3A199%2C36

### Bug description

When we add thumbs swiper after the parent swiper has been initialized i.e. `parentSlider.thumbs.swiper = thumbSlider`, they are linked together and work as expected except for clicking on the thumbnails to slide to that slide. As we cannot enable slideToClickedSlide when thumbs module is being used, that leaves us no easy way to get the expected behaviour from the thumbs slider.

### Expected Behavior

Adding thumbnail slider after the parent slider has been instantiated should work similar to the case when the thumbs slider is specified in the original params during initialization.

### Swiper version

9.2.4

### Platform/Target and Browser Versions

Chrome 115.0.5790.102 on Windows, MacOS and Linux

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

**Comment by Shulaolao** [comment]
I also encountered this problem in nuxt3 +swiper 10.2.0. When I refreshed the page, clicking the thumbnails cannot slide. I think the thumbnails loaded earlier than the large swiper.
![original codes](https://github.com/nolimits4web/swiper/assets/99474694/be020423-a025-4c32-98bf-8d11f12fef67)
 Cause thumbnails to trigger swiper event without setting swiper parameter for large swiper, So I added a <client-only> wrapper to the large swiper to make the thumbnails render after the large swiper is loaded and rendered to ensure that the swiper event works
![work successfully codes](https://github.com/nolimits4web/swiper/assets/99474694/03b1a8ea-d2d4-4232-9189-784996a20ff4)
